pyproject.toml

```diff
@@ -90,6 +91,7 @@ mkdocs-extensions = "^0.1.2"
 pymdown-extensions = "^10.0.1"
 mkdocs-material = "^9.1.13"
 mkdocstrings = {extras = ["python"], version = "^0.21.2"}
+jupyter = "^1.0.0"
```

Comment:
Im getting a very odd error with docker. If I run `garden-ai docker check` it throws the error:
Garden can't access Docker on your computer.
This doesn't look like one of the typical error cases. Printing error from Docker:
Error while fetching server API version: Not supported URL scheme http+docker

As suggested [here](https://github.com/geerlingguy/ansible-role-docker/issues/462), I downgraded from requests==2.32.3 to requests==2.31.0 and that seemed to fix it for me. Id say pin requests to 2.31.0 for now just to be safe.